src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1432,6 +1436,42 @@ QString MainWindow::get_filter_string(const int filter)
     return filters.join(";;");
 }
 
+void MainWindow::initialize_ocio()
+{
+    // Try first a user specified OCIO config.
+    if (getenv("OCIO"))
+    {
+        try
+        {
+            m_ocio_config = OCIO::GetCurrentConfig();
+            RENDERER_LOG_INFO("Using OpenColorIO config: %s", getenv("OCIO"));
+            return;
+        }
+        catch (const OCIO::Exception&)
+        {
+        }
+    }
+
+    // Try the bundled default OCIO config.
+    const bf::path root_path(Application::get_root_path());
+    const string default_ocio_config = (root_path / "ocio" / "config.ocio").string();
+
+    try
+    {
+        m_ocio_config = OCIO::Config::CreateFromFile(default_ocio_config.c_str());
+        RENDERER_LOG_INFO("Using OpenColorIO config: %s", default_ocio_config.c_str());
+        OCIO::SetCurrentConfig(m_ocio_config);
+        return;
+    }
+    catch(...)
```

Comment:
`catch (...)`

Also, this is too wide a net, be more specific in which exceptions you intend to catch.